bundles/org.openhab.binding.teslascope/src/main/java/org/openhab/binding/teslascope/internal/api/ChargeState.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.teslascope.internal.api;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Class for holding the set of parameters used to read the controller variables.
+ *
+ * @author Paul Smedley - Initial Contribution
+ *
+ */
+@NonNullByDefault
+public class ChargeState {
+    // charge_state
+    public int battery_level;
```

Comment:
```suggestion
    @SerializedName("battery_level")
    public int batteryLevel;
```
This should be applied to all dto classes as naming convention also applies here. Sorry, this is not the most fun task. I have no knowledge of a tool that generates the4se json classes with this annotation, so i can't advice on a streamlined workflow here.

Reply:
I was wondering what the solution was for this, as I'd noticed the warnings. You're right, not fun, but it's a one-off activity. I'll try get it cleaned up this evening (my time)

Reply:
I've  done this for ChargeState.java but I'm still seeing:
Checkstyle	3	28	style	MemberNameCheck	Name 'battery_level' must match pattern '^[a-z][a-zA-Z0-9]*$'.

Reply:
Fixed.